tests/test_10_use_cases.py

```diff
@@ -0,0 +1,210 @@
+        ## End of verification code
+        assert data[key]["status"] == "down"
+
+        # set one link to up
+        self.net.net.configLinkStatus('Tenet01', 'Tenet03', 'up')
+
+        time.sleep(15)
+
+        data = requests.get(api_url).json()
+        key = next(iter(data))
+        assert data[key]["status"] == "up"
+
+        # test connectivity
+        assert ', 0% packet loss,' in h6.cmd('ping -c4 10.1.1.8')
+
+    def test_011_send_topo_update_with_intra_domain_link_down_path_not_found(self):
+        ''' Use case 1
+            OXPO sends a topology update with an intra-domain link down.
+            When not possible, the SDX Controller will only update the L2VPN’s status using the means available.
+
+            - Create a L2VPN between Tenet01:50 and Tenet02:50
+            - Check if that L2VPN status is UP (some wait may be required here)
+            - Run a PING test between hosts to make sure we have connectivity
+            - Config the Link Tenet01 — Tenet02 to down (some wait may be required here)
+            - Query the L2VPN and check the status to validate it is UP (SDX should find another path using links from SAX)
+            - Config the Link Tenet01 — Tenet02 to UP (some wait may be required here)
+            - Query the L2VPN and check the status to validate it continue to be UP 
+            - Run a PING test between hosts to make sure we have connectivity
+        '''
+        api_url = SDX_CONTROLLER + '/l2vpn/1.0'
+        payload = {
+            "name": "Test L2VPN creation",
+            "endpoints": [
+                {"port_id": "urn:sdx:port:tenet.ac.za:Tenet01:50","vlan": "100"},
+                {"port_id": "urn:sdx:port:tenet.ac.za:Tenet02:50","vlan": "100"}
+            ]
+        }
+        response = requests.post(api_url, json=payload)
+        assert response.status_code == 201, response.text
+
+        # wait until status changes for under provisioning to up
+        time.sleep(5)
+
+        data = requests.get(api_url).json()
+        assert len(data) == 1
+        key = next(iter(data))
+        assert data[key]["status"] == "up"
+
+        h6, h7 = self.net.net.get('h6', 'h7')
+        h6.cmd('ip link add link %s name vlan100 type vlan id 100' % (h6.intfNames()[0]))
+        h6.cmd('ip link set up vlan100')
+        h6.cmd('ip addr add 10.1.1.6/24 dev vlan100')
+        h7.cmd('ip link add link %s name vlan100 type vlan id 100' % (h7.intfNames()[0]))
+        h7.cmd('ip link set up vlan100')
+        h7.cmd('ip addr add 10.1.1.7/24 dev vlan100')
+
+        # test connectivity
+        assert ', 0% packet loss,' in h6.cmd('ping -c4 10.1.1.7')
+
+        # set one link to down
+        self.net.net.configLinkStatus('Tenet01', 'Tenet02', 'down')
+
+        time.sleep(15)
+
+        # SDX should find another path using links from SAX
+        data = requests.get(api_url).json()
+        key = next(iter(data))
+        assert data[key]["status"] == "up"
+
```

Comment:
Please add here another check to validate connective is still good (ping from h6 to h7 and assert with `0% packet loss`